requirements.txt

```diff
@@ -1 +1,2 @@
-https://github.com/Rapptz/discord.py/archive/rewrite.zip#egg=discord.py[voice]
\ No newline at end of file
+git+https://github.com/Rapptz/discord.py.git@rewrite#egg=discord.py[voice]
```

Comment:
Why are you changing this?

Reply:
It's the preferred way of referring to a git repository in the requirements.txt file:
https://pip.readthedocs.io/en/stable/reference/pip_install/#git